packages/opencensus-web-core/src/trace/model/tracer-base.ts

```diff
@@ -52,6 +52,9 @@ export class TracerBase implements webTypes.TracerBase {
    */
   readonly activeTraceParams = {};
 
+  /** A configuration for starting the tracer */
+  private config!: webTypes.TracerConfig;
```

Comment:
Is there a default value we can give to this? Maybe just an empty object so `rivate config: webTypes.TracerConfig = {}`?